README.md

```diff
@@ -1 +1,174 @@
+-   [👥 Authors](#authors)
+-   [🎥 Live Demo](#live-demo)
+-   [🎥 Project Presentation](#Project-presentation)
+-   [🤝 Contributing](#contributing)
+-   [⭐️ Show your support](#support)
+-   [🙏 Acknowledgments](#acknowledgements)
+-   [📝 License](#license)
+
+
+# 📖 My Personal Portfolio <a name="about-project"></a>
+
+> This website shows a sneak peek into the projects I have developed and deployed, showcasing my skills in HTML and CSS.With a passion for web development, I have honed my expertise in these technologies to create stunning and functional websites.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>HTML</summary>
+  <ul>
+    <li><a href="https://www.w3schools.com/html/">Html</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>CSS</summary>
+  <ul>
+    <li><a href="https://www.w3schools.com/css/default.asp">css</a></li>
+  </ul>
+</details>
+
+
+### Key Features <a name="key-features"></a>
+
+- Use the Figma design for the project😊👌
+- Use the semantics elements💯
+- Use the validation HTML for the form .🚀
+- Use Form spree for the contact form 😃👍
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+### Prerequisites
+
+In order to run this project you need:
+
+- A web browser to view output e.g [Google Chrome](https://www.google.com/chrome/).
+- An IDE e.g [Visual studio code](https://code.visualstudio.com/).
+- Install the `npm` package manager use this [to install both node and npm](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm).
+- [A terminal](https://code.visualstudio.com/docs/terminal/basics).
+
+
+### Setup
+
+Clone this repository to your desired folder or download the Zip folder:
+
+```
+git clone https://github.com/sheezu900/module1-capstone-project.git
+```
+
+- Navigate to the location of the folder in your machine:
+
+**``you@your-Pc-name:~$ cd desktop``**
+
+### Install
+
+To install all dependencies, run:
+
+```
+npm install
+```
+
+### Usage
+
+To run the project, follow these instructions:
+
+- After Cloning this repo to your local machine.
+- To get it running on your default browser and local host, run:
+
+1. Open the index with live server from the code editor
+2. Or you can double click on the index to open it in the browser
+
+### Run tests
+
+To run tests, run the following command:
+
+- Track HTML linter errors run:
+```
+npx hint .
+```
+- Track CSS linter errors run:
+```
+npx stylelint "**/*.{css,scss}"
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Shah Zaman**
+
+-   GitHub: [@githubhandle](https://github.com/sheezu900/)
+-   LinkedIn: [@Linkedin](https://www.linkedin.com/in/shahzaman900/)
+
+<p align="right">(<a href="#live-demo">back to top</a>)</p>
+
+## 🎥 Live Demo <a name="authors"></a>
+
+-   Live Demo: [@LiveDemo](https://sheezu900.github.io/module1-capstone-project/)
+
```

Comment:
- [ ] You did great deploying the project and adding the live link, :+1: :+1:  But looking at the video below, you can see the deployed version is not responding or it doesn't have the latest version of your project. I'll suggest you update this as Microverse required your live version to also be up-to-date and functional. :+1: :wave: 

- - VIDEO REFERENCE:

https://github.com/sheezu900/module1-capstone-project/assets/107632780/efac26f9-ebb3-4089-8017-390b9f3ae76c